modules/persistent-storage-csi-smb-cifs-create-sc.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-smb-cifs-create-sc_{context}"]
+= Creating a storage class for SMB CIFS
+
+After installing the Operator, you should create a storage class for dynamic provisioning of Server Message Block (SMB) protocol Common Internet File System (CIFS) dialect volumes.
+
+.Prerequisites
+* You are logged in to the running {product-title} cluster.
+
+.Procedure
+To create a storage class:
+
+* Create a storage class using the following example YAML file:
++
+[source,yaml]
+.Example YAML file
+--
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: samba
+provisioner: smb.csi.k8s.io
+parameters:
+  source: //samba-server.samba-server.svc.cluster.local/share
```

Comment:
I'd suggest some explanation for this line:

> This assumes that Samba server is installed somewhere and reachable from the cluster by the hostname `samba-server.samba-server.svc.cluster.local`. And the server is configured to have the path `/share` among exported shares.

Reply:
yes +1 let's be exhaustive on the pre-requisites